index.html

```diff
@@ -155,9 +155,11 @@
 enables verifiable, decentralized digital identity. A <a>DID</a> identifies
 any subject (e.g., a person, organization, thing, data model, abstract entity, etc.)
 that the controller of the <a>DID</a> decides that it 
-identifies. These new identifiers are designed to enable the controller 
-of a <a>DID</a> to prove control over it and to be implemented independently 
-of any centralized registry, identity provider, or certificate authority. 
+identifies.
+DIDs are implemented
+independently of any centralized registry, identity provider, or
```

Comment:
This removes the hard fought language "are designed to enable" that created space for DID method creators to make choices and replaces it with simply "are implemented...". The WG was unable to get started without this important distinction and I would expect this language change to be rejected.